hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -372,9 +373,15 @@ private void reloadTrigger(Trigger trigger) {
         if (log.isDebugEnabled()) {
             log.debug("Reloading " + trigger);
         }
+
         // Look for the Trigger in the rules engine, if it is there then remove everything about it
         removeTrigger(trigger);
 
+        // If the trigger is not enabled then just stop after ensuring it's no longer loaded
```

Comment:
+1, I should have run the cluster scenario.  The cluster fails because the enabled flag would always be false before being fetched, if the trigger was set to only tenantId+triggerId.